libs/ui/packages/react/src/pre-ldls/components/VirtualList/VirtualList.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { FixedSizeList as List } from "react-window";
+
+export const VirtualList = ({
+  children,
+  itemHeight,
+  maxHeight,
+}: {
+  children: React.ReactNode[];
+  itemHeight: number;
+  maxHeight: number;
+}) => {
+  const childrenLength = Array.isArray(children) ? children.length : 0;
```

Comment:
Why `Array.isArray(children)` children is needed ? isn't it enforced by the type of children ?